drivers/base/nmidebug/nmidebug.c

```diff
@@ -13,49 +12,31 @@
 
 /* FUNCTIONS ******************************************************************/
 
-PCHAR NmiBegin = "NMI4NMI@";
-
-FORCEINLINE
-VOID
-NmiClearFlag(VOID)
-{
-    ((PCHAR)&KiBugCheckData[4])[0] -= (NmiBegin[3] | NmiBegin[7]);
```

Comment:
`NmiBegin[3] == '4' == 0x34` and `NmiBegin[7] == '@' == 0x40`, thus `NmiBegin[3] | NmiBegin[7] == 0x74 == 't'`

Reply:
It does seem correct as is.

I was not aware of win2k3 hyper-v, so this is interesting behavior for sure.